peregrine/acquisition.py

```diff
@@ -551,6 +551,38 @@ def __str__(self):
   def __repr__(self):
     return "<AcquisitionResult %s>" % self.__str__()
 
+  def __eq__(self, other):
+    return self._equal(other)
+
+  def __ne__(self, other):
+    return not self._equal(other)
+
+  def _equal(self, other):
+    """
+    Compare equality between self and another :class:`AcquisitionResult` object.
+
+    Parameters
+    ----------
+    other : :class:`AcquisitionResult` object
+      The :class:`AcquisitionResult` to test equality against.
+
+    Return
+    ------
+    out : bool
+      True if the passed :class:`AcquisitionResult` object is identical.
+    
+    """
+    if set(self.__dict__.keys()) != set(other.__dict__.keys()):
+      return False
+
```

Comment:
This looks fine, if you need some kind of floating equality check of dictionaries there's a function for that in https://github.com/swift-nav/sbp_log_analysis/blob/master/sbp_log_analysis/utils.py.